contracts/test/MockUtilityToken.sol

```diff
@@ -0,0 +1,107 @@
+pragma solidity ^0.5.0;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Auxiliary chain: MockUtilityToken
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+
+import "../gateway/UtilityToken.sol";
+
+/**
+ *  @title MockUtilityToken contract.
+ *
+ *  @notice This is for testing purpose.
+ *
+ */
+contract MockUtilityToken is UtilityToken {
```

Comment:
Maybe call this a spy and not a mock?

* https://www.martinfowler.com/bliki/TestDouble.html
* https://en.wikipedia.org/wiki/Test_double

/cc @deepesh-kn @sarvesh-ost 

Should we clean up our test doubles naming in a separate ticket?